api/handler.go

```diff
@@ -94,8 +94,9 @@ func NewHandler(
 	infoServer := infoserver.NewServer(logger, version, workerVersion, credsManagers)
 
 	handlers := map[string]http.Handler{
-		atc.GetConfig:  http.HandlerFunc(configServer.GetConfig),
-		atc.SaveConfig: http.HandlerFunc(configServer.SaveConfig),
+		atc.GetConfig:                 http.HandlerFunc(configServer.GetConfig),
+		atc.SaveConfig:                http.HandlerFunc(configServer.SaveConfig),
+		atc.SaveConfigSkipCredentials: http.HandlerFunc(configServer.SaveConfigSkipCredentials),
```

Comment:
(Already mentioned above but I think this should just be a query param on the existing endpoint.)